src/components/feed/post/minified.tsx

```diff
@@ -0,0 +1,35 @@
+// Modules
+import { format } from "date-fns";
+
+// Components
+import PostTitle from "@components/feed/title";
+import Title from "@components/global/title";
+
+// Types
+import { MinifiedPost as MinifiedPostType } from "@utils/types/feed/post";
+
+const MinifiedPost = ({ post }: { post: MinifiedPostType }) => {
+  return (
+    <li className="hover:bg-light-primary-0.08-tlc">
+      <button className="card container-surface text-left">
+        <Title
+          icon={
+            <div
+              className="grid place-items-center w-10 h-10 aspect-square
+                text-light-on-secondary font-medium bg-light-secondary rounded-full"
+            >
+              {post.author ? post.author[0].toUpperCase() : ":)"}
```

Comment:
You checked if the array exist but you did not check its length so if an empty array is inputted then it would still not work